packages/peregrine/lib/Router/magentoRouteHandler.js

```diff
@@ -28,19 +28,6 @@ export default class MagentoRouteHandler extends Component {
         }
     };
 
-    // TODO: Add the ability to customize the cache name
-    async addToCache(urls) {
-        if (!window.caches) {
-            throw new Error(
-                'Current environment does not support CacheStorage at window.caches.'
-            );
-        }
-        const myCache = await window.caches.open(
-            `workbox-runtime-${location.origin}/`
-        );
-        await myCache.addAll(urls);
-    }
-
```

Comment:
Is this code obsolete now or did it move?

Reply:
Yeah I had a discussion with the wider team and they said this was part of some old workbox runtime which was since refactored out, but this legacy code was retained. So we agreed to remove it here.